source/extensions/filters/http/composite/filter.cc

```diff
@@ -20,6 +20,12 @@ RValT delegateFilterActionOr(FilterPtrT& filter, FuncT func, RValT rval, Args&&.
 
   return rval;
 }
+
+// Own version of lambda overloading since std::overloaded is not available to use yet.
+template <class... Ts> struct Overloaded : Ts... { using Ts::operator()...; };
+
+template <class... Ts> Overloaded(Ts...) -> Overloaded<Ts...>;
```

Comment:
I think this has been defined in several places in the repo with different names, might be nice to have this in a shared location but not necessary for this PR, I'll file an issue

Reply:
Sure, I have filed #26442 and assigned to myself.

Reply:
Thanks!